docs/tracing/05-01-jaeger.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Jaeger chart
+type: Configuration
+---
+
+To configure the Jaeger chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>**TIP:** To learn more about how to use overrides in Kyma, see the following documents:
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Top-level charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-top-level-charts-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **jaeger.memory.maxTraces** | Defines the maximum amount of traces that Jaeger can store. | `40000` |
+| **resources.limits.memory** | Defines limits for memory resources. | `512M` |
```

Comment:
Somehow it does not reflect that Jager stores all traces in memory and that setting will restrict the maximum size of the storage. It will fail maxTraces will not fit into the memory.

Reply:
I updated the description. Hope it is clear now.